google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java

```diff
@@ -25,13 +25,10 @@
 import com.google.api.gax.rpc.StreamController;
 import com.google.cloud.Timestamp;
 import com.google.cloud.firestore.v1.FirestoreSettings;
-import com.google.firestore.v1.RunAggregationQueryRequest;
-import com.google.firestore.v1.RunAggregationQueryResponse;
-import com.google.firestore.v1.RunQueryRequest;
-import com.google.firestore.v1.StructuredAggregationQuery;
-import com.google.firestore.v1.Value;
+import com.google.firestore.v1.*;
+import com.google.firestore.v1.StructuredAggregationQuery.Aggregation;
 import com.google.protobuf.ByteString;
-import java.util.Set;
+import java.util.*;
```

Comment:
`hashCode` and `equals` inside AggregateQuery needs to be changed :)

Reply:
Done.